src/components/containers/Section.js

```diff
@@ -21,12 +24,23 @@ export default class Section extends Component {
   }
 
   processAndSetChildren(nextProps, nextContext) {
+    const {fullContainer} = nextContext;
+    const {localize: _} = nextProps;
     this.sectionVisible = false;
 
     const children = React.Children.toArray(nextProps.children);
     this.children = [];
     let menuPanel = null;
 
+    /*
+     * show opacity disabled message for scatter traces that have a fill 'tonexty' || 'tonextx'
+     * based on:
+     * https://github.com/plotly/plotly.js/blob/master/src/traces/scatter/clean_data.js#L13
+     */
+    const cannotSetTraceOpacity =
+      fullContainer.type === 'scatter' &&
+      (fullContainer.fill !== 'none' || fullContainer.fill !== 'toself');
+
```

Comment:
@alexcjohnson is `cannotSetTraceOpacity` correct?

I feel like since there's an accordion menu for each trace in the editor, the rest of this logic doesn't apply: 
https://github.com/plotly/plotly.js/blob/master/src/traces/scatter/clean_data.js#L24

Reply:
if we're looking at the trace from `fullData`, we should just be able to use the results of `scatter/clean_data`, ie `fullContainer.type === 'scatter' && fullContainer.opacity === undefined`, right? Because the multi-trace aspect of it is really important for alleviating confusion and you're right that you don't want to try and reconstruct that here.

Reply:
thanks! will do that